asapdiscovery/simulation/scripts/MProFunction.py

```diff
@@ -0,0 +1,239 @@
+import sys
+from pathlib import Path
+import multiprocessing as mp
+
+# Configure logging
+import logging
+from rich.logging import RichHandler
+FORMAT = "%(message)s"
+from rich.console import Console
+logging.basicConfig(
+    level=logging.INFO, format=FORMAT, datefmt="[%X]", handlers=[RichHandler(markup=True)]
+)
+log = logging.getLogger("rich")
+
+## Preliminary Imports
+import os
+import re
+import argparse
+import openmm
+import rdkit
+from rdkit import Chem
+
+## OpenMM Imports
+from openmm.app import PDBFile
+from openmm import unit, app
+from openmm.app import PME, HBonds
+from openmm.app import Modeller
+from openmm import LangevinMiddleIntegrator
+from openmm.app import Simulation
+from openmm import MonteCarloBarostat
+from openmm import Platform
+from openmm.app import DCDReporter
+from openmm.app import StateDataReporter
+from openmm.app import Simulation
+import mdtraj
+from mdtraj.reporters import XTCReporter
+from openff.toolkit.topology import Molecule
+from openmmforcefields.generators import SystemGenerator
+
+#Simulation Parameters
+
+steps = 10000
+equilibrationSteps = 100
+reportingInterval = 1250
+dt = 4.0 * unit.femtoseconds
+temperature = 300 * unit.kelvin
+friction = 1.0 / unit.femtoseconds
+pressure = 1.0 * unit.atmospheres
+barostatInterval = 25
+nonbondedMethod = PME
+nonbondedCutoff = 1.0*unit.nanometers
+ewaldErrorTolerance = 0.000001
+constraints = HBonds
+rigidWater = True
+constraintTolerance = 0.000001
+hydrogenMass = 4.0*unit.amu
+platform = Platform.getPlatformByName('CUDA')
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+
+    ## Input arguments
+    parser.add_argument(
+        "-i",
+        "--input_dir",
+        type = str,
+        default = "/data/chodera/asap-datasets/current/sars_01_prepped_v3/",
+        help = "Path to PDB and SDF files"
+    )
+    parser.add_argument(
+        "-p", 
+        "--protein", 
+        type = str,
+        default = "/data/chodera/lemonsk/asap-datasets/openmm_setup_processed/prepped_receptor_0-processed.pdb",
+        help = "Path to PDB file to simulate"
+    )
+    parser.add_argument(
+        "-l", 
+        "--ligand",
+        type = str,
+        default = "/data/chodera/lemonsk/covid-moonshot-ml/asapdiscovery/simulation/tests/inputs/MAT-POS-f2460aef-1.sdf",
+        help="Ligand"
+    )
```

Comment:
Instead of these arguments, you should have strings for globbing! 
i.e
```
parser.add_argument(
        "-p", 
        "--protein_glob", 
        type = str,
        default = "*/*protein.pdb",
        help = "String to correctly glob protein file using input directory"
    )
    parser.add_argument(
        "-l", 
        "--ligand_glob",
        type = str,
        default = "*/*.sdf",
        help="String to correctly glob ligand file using input directory"
    )
```